### DISCUSSION TITLE
Custom color

### DISCUSSION THREAD

**Discussion by pizzav-xyz** [end-4/dots-hyprland#674] [Q&A] [answered]
How do I make it use a custom color instead of wallpaper

**Comment by end-4** [comment, marked as answer]
in launcher type `>color` then pick the desired color on your screen

**Reply by pizzav-xyz** [reply]
like the super key?

**Reply by end-4** [reply]
yes you hit super to open the launcher, and from there do that

**Comment by pizzav-xyz** [comment]
@end-4 this doesnt seem to work anymore
